wagtail/admin/templates/wagtailadmin/pages/choose_parent.html

```diff
@@ -0,0 +1,31 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n wagtailadmin_tags%}
+
+{% block titletag %}Choose a parent page{% endblock %}
+
+{% block extra_css %}
+    {{ block.super }}
+    {{ form.media.css }}
+{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+    {% include "wagtailadmin/pages/_editor_js.html" %}
+    {{ form.media.js }}
+{% endblock %}
+
+{% block content %}
+    {% block header %}
+        {% include "wagtailadmin/shared/header.html" with title=_("Create a new ") subtitle=model_name icon="doc-empty-inverse" %}
+    {% endblock %}
```

Comment:
I can't explain the details right now, but ideally we want to move away from `header.html` in favour of `slim_header.html` with the breadcrumbs. See how we render the `wagtail.admin.views.generic.models.CreateView` and its corresponding `wagtailadmin/generic/form.html` template for example.

```suggestion
{% block main_content %}
```